source/_posts/Applying-Proguard-in-an-android-application.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Applying Proguard in an android application
+date: 2020-07-27 18:40:48
+author: Diwakar Singh
+category: Android
+tags:
```

Comment:
Content   should have link to previous blog

Reply:
Added